### DISCUSSION TITLE
Getting Started with Data Science

### DISCUSSION THREAD

**Discussion by BekahHW** [deepgram/community#41] [General help]
We love sharing our learning resources. What are your favorite ways to get started with Data Science? A couple that are on our list are:
- [Data Science](https://www.coursera.org/specializations/jhu-data-science)  
- [Redirect](https://nbdev.fast.ai/tutorial.html) 
- [Learning Resources - Data Engineering Wiki](https://dataengineering.wiki/Learning+Resources)

**Comment by briancbarrow** [comment]
FreeCodeCamp is always the first place I look when learning new things. https://www.freecodecamp.org/news/data-science-learning-roadmap/

**Reply by briancbarrow** [reply]
Here's another video. It is a couple of years old but I imagine the principles still apply https://www.youtube.com/watch?v=XU5pw3QRYjQ

**Comment by dotAadarsh** [comment]
Microsoft offers a 10-week, 20-lesson curriculum all about Data Science - [Data Science for Beginners](https://microsoft.github.io/Data-Science-For-Beginners/#/)
